packages/components/src/components/relative-datepicker/relative-datepicker.tsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { toMoment, daysFromTodayTo } from '@deriv/shared';
 import Text from '../text';
 
-const RelativeDatepicker = ({ onChange, min_date, max_date, title }) => {
-    const hidden_input_ref = React.useRef();
+type TRelativeDatepickerProps = {
+    max_date?: number | string;
+    min_date?: number | string;
```

Comment:
@jim-deriv in duration-mobile.jsx we pass `min` and `max` into this component but we expect `max_date` and `min_date` here. Could you please check if that's right?

Reply:
I see the errors with this fields on local host in console, when I go to account personal details.
Someone is really weird here...

Reply:
it seems, that we have no error on production, because of internal assignments of min and max values for input.